lifelines/fitters/cox_time_varying_fitter.py

```diff
@@ -801,7 +801,7 @@ def _compute_cumulative_baseline_hazard(self, tv_data, events, start, stop, weig
             hazards = self.predict_partial_hazard(tv_data).values
 
         unique_death_times = np.unique(stop[events.values])
-        baseline_hazard_ = pd.DataFrame(np.zeros_like(unique_death_times), index=unique_death_times, columns=["baseline hazard"])
+        baseline_hazard_ = pd.DataFrame(np.zeros_like(unique_death_times).astype(float), index=unique_death_times, columns=["baseline hazard"])
```

Comment:
The warning posted says `Value '0.6058777002252564' has dtype incompatible with int64`, but this is the `zeros_like` function - which should produces `0`. I'm confused.

Reply:
Yeah, because it (a couple lines down) tries to assign something to this column.